src/utils/profile.ts

```diff
@@ -0,0 +1,17 @@
+import firebase from "firebase/app";
+
+export const updateProfileEnteredVenueIds = async (
+  prevEnteredVenueIds: string[] | undefined,
```

Comment:
```suggestion
  prevEnteredVenueIds: readonly string[] | undefined,
```

Which I expect will break your `enteredVenueIds.push(venueId)` line. We want non-mutating functions like `array.concat` instead

- https://lorenstewart.me/2017/01/22/javascript-array-methods-mutating-vs-non-mutating/#iiaddnonmutating